### DISCUSSION TITLE
Language Filter

### DISCUSSION THREAD

**Discussion by pr0xyfl00d3r** [sct/overseerr#560] [Feature Requests]
I have been looking through the UI and one thing comes to me about the Language of the Movie or Series you don't know if its of your native language or acceptable language until you view the details, I think maybe a filter showing what language you want, and maybe show it on the poster page as you browse

**Comment by sct** [comment]
I think this should give you what you need https://github.com/sct/overseerr/issues/375

**Reply by pr0xyfl00d3r** [reply]
That sounds spot on, keep up the great work looks fantastic
